src/System.Diagnostics.Process/src/Microsoft/Win32/SafeHandles/SafeThreadHandle.Unix.cs

```diff
@@ -19,17 +18,18 @@ namespace Microsoft.Win32.SafeHandles
 {
     internal sealed partial class SafeThreadHandle : SafeHandle
     {
-        private const int DefaultInvalidHandleValue = -1; // TODO: Determine this
+        private const int DefaultInvalidHandleValue = -1;
 
         public override bool IsInvalid
         {
             [SecurityCritical]
-            get { throw NotImplemented.ByDesign; } // TODO: Implement this
+            get { return (long)handle < 0; }
         }
 
         protected override bool ReleaseHandle()
         {
-            throw NotImplemented.ByDesign; // TODO: Implement this
+            // Nop.
```

Comment:
Actually, it turns out that SafeThreadHandle isn't used at all in the Unix implementation.  I've recombined the Windows partial types, deleted the Unix one, and removed it from the Unix build entirely.